### PR TITLE
Add tinyxml2 (6.2.0-p1)

### PR DESCRIPTION
**Pull request by lsolanka**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/lsolanka/hunter
  * https://travis-ci.org/lsolanka/hunter/builds/375252643

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/241

### CODE REVIEW

File: docs/packages/pkg/tinyxml2.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.. _pkg.tinyxml2:
+
+TinyXML2
+========
+
+-  `Official <https://github.com/leethomason/tinyxml2>`__
+-  `Hunterized <https://github.com/hunter-packages/tinyxml2>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/tinyxml2/CMakeLists.txt>`__
+-  Added by `Lukas Solanka <https://github.com/lsolanka>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/tinyxml2.rst]: `N` -> `1426`

Comment (lsolanka) [reply docs/packages/pkg/tinyxml2.rst]: yes, sorry :-); done!

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3846
* https://travis-ci.org/ingenue/hunter/builds/375286378?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3847
* https://travis-ci.org/ingenue/hunter/builds/375289337?utm_source=github_status&utm_medium=notification

**Pull request merged**